CHANGELOG.md

```diff
@@ -9,6 +9,7 @@ Try to keep listed changes to a concise bulleted list of simple explanations of
 ## [Unreleased]
 ### Changed
 - Make experimental reverse_expand behave the same as old reverse_expand in case of timeouts. [#2649](https://github.com/openfga/openfga/pull/2649)
+- Increase number of querying goroutines in experimental reverse_expand to `resolveNodeBreadthLimit`. [#2652](https://github.com/openfga/openfga/pull/2652)
```

Comment:
Maybe move this to `### Added` and state that the number of querying goroutines is now configurable through the `resolveNodeBreadthLimit` configuration